docs/schema.json

```diff
@@ -2149,15 +2156,17 @@
         },
         "finalPriceConsent": {
           "typeRef": "#/domains/Generic/types/PriceConsent",
-          "description": "Client's consent for final price, should exactly match the <code>finalPrice</code> object from output.<br/>Automation will not proceed with placing order until the consent is provided."
+          "description": "Client's consent for final price, should exactly match the <code>finalPrice</code> object from output.<br/>Automation will not proceed with placing order until the consent is provided.",
+          "fooBarBaz": "finalPriceConsent"
         },
         "options": {
           "typeRef": "#/domains/VacationRental/types/Options",
           "default": {},
           "initial": true
         },
         "deposit": {
```

Comment:
naming: in some cases `selected` part missing so again, we can't rely on `selected-available` in order to build Guided Docs UI